publishable/lang/fr/database.php

```diff
@@ -51,30 +44,31 @@
     'success_update_table'      => 'Table :table mise à jour avec succès',
     'table_actions'             => 'Actions sur le tableau',
     'table_columns'             => 'Colonnes de table',
-    'table_has_index'           => 'La table comporte déjà un indice primaire (primary index).',
+    'table_has_index'           => 'La table comporte déjà un index primaire (primary index).',
     'table_name'                => 'Nom de la table',
-    'table_no_columns'          => 'La table n\a pas de colonnes...',
+    'table_no_columns'          => 'La table n\'a pas de colonnes...',
     'type'                      => 'Type',
     'type_not_supported'        => 'Type non supporté',
     'unique'                    => 'UNIQUE',
     'unknown_type'              => 'Type inconnu',
     'update_table'              => 'Mettre la table à jour',
     'url_slug'                  => 'URL Slug (doit être unique)',
-    'url_slug_ph'               => 'URL slug (exemple : articles)',
+    'url_slug_ph'               => 'URL Slug (exemple : articles)',
     'visibility'                => 'Visibilité',
     'relationship'              => [
         'relationship'         => 'Relation (Relationship)',
         'relationships'        => 'Relations (Relationships)',
-        'has_one'              => 'En a un (Has One)',
-        'has_many'             => 'En a pluieurs (Has Many)',
-        'belongs_to'           => 'Appartient à (Belongs To)',
-        'belongs_to_many'      => 'Appartient à pluieurs (Belongs To Many)',
-        'which_column_from'    => 'Quelle colonne du',
-        'is_used_to_reference' => 'est utilisée pour référencer le',
-        'pivot_table'          => 'Tableau croisée dynamique (Pivot Table)',
+        'has_one'              => null,
```

Comment:
Any particular reason these are null?

Reply:
Yes, because it shouldn't be translated...

Reply:
@emptynick tested it and it looks like null value and missing key behave the same.  Can you verify this and remove the entry? I feel the null value may get confusing and difficult to maintain long-term.  I'd rather have it be missing entirely, which indicates it needs a translation (or should be left alone)

Reply:
For me, not having a value or value set on Null use the default translation (EN). That's weird.
Anyway I commit the translation of those values in EN.